QUEST.tsv

```diff
@@ -2436,13 +2436,13 @@ QUEST_20150918_002443	There is a box over there. Shall we check it?
 QUEST_20150918_002444	$Let's check that barrel over there.
 QUEST_20150918_002445	Is that a wing of a Goddess Statue over there?
 QUEST_20150918_002446	Huh? How did you get up here?
-QUEST_20151001_002447	Amazing.{nl}It must be created hundreds of years ago..{nl}It seems it speaks by watching you.{nl}
-QUEST_20151001_002448	But, I don't feel any evil energy nor the determination.{nl}So then, this Revelation is really of Goddess Laima's... That's right.{nl}
-QUEST_20151001_002449	We are more surprised about the savior.{nl}
-QUEST_20151001_002450	Among many Revelators, the one who can save this world and the Goddesses...{nl}The Revelation says that you are the savior.{nl}
-QUEST_20151001_002451	You better not tell the world about it.{nl}If you don't want the demons to go look for you.{nl}
-QUEST_20151001_002452	The Revelation doesn't want me to mention about the savior, I guess.{nl}Okay, please go back to meet Uska.{nl}
-QUEST_20151001_002453	He will tell you where to go.{nl}May the Goddess bless you in the future.
+QUEST_20151001_002447	Amazing.{nl}It must have been created hundreds of years ago..{nl}It seems it speaks of you.{nl}
+QUEST_20151001_002448	I don't feel any evil energy nor malicious intent.{nl}Then, this revelation must have come from the Goddess Laima...{nl}
+QUEST_20151001_002449	I'm suprised about this talk of a savior.{nl}
+QUEST_20151001_002450	Amongst the many Revelators, the one who can save the world and the Goddesses...{nl}The revelation says you are the savior.{nl}
+QUEST_20151001_002451	You better keep quiet about this.{nl}Unless you want the demons looking for you.{nl}
+QUEST_20151001_002452	The revelation doesn't want me to mention anything about the savior.{nl}Okay, please hurry back to Uska.{nl}
+QUEST_20151001_002453	He'll know where to go.{nl}May the Goddess bless you.
```

Comment:
"He'll know where to go" is ambiguous. "He will tell you where to go" is fine and doesn't need to be changed. 

Why did you remove 'in the future'? It's there.